service-commands/scripts/setup-instance.sh

```diff
@@ -150,6 +111,14 @@ case "$service" in
 			# TODO fix install and provisioning for fast
 		fi
 
+		# configure local files: /etc/hosts, /etc/ssh/ssh_config.d/60-audius.conf
+		configure_etc_hosts
+		set_ssh_serveralive
+
+		# upload personal files: ~/.gitconfig, ~/.zshrc, ~/.zshenv, ~/.p10k.zsh
+		upload_gitconfig
+		setup_zsh
```

Comment:
@csjiang I saw you were having issues with this on your previous run. Going to test once more on this fresh branch and see if I can reproduce before running `setup_zsh()` again.

Reply:
I just verified that zsh worked. Would you mind giving it a shot on your side? Hopefully it was just a glitch at install time with zsh.

```
NAME=joaquinXX
./service-commands/scripts/A setup --protocol-git-ref jc-setup-docs.3 remote-dev ${NAME}-remote-dev
```

Reply:
Works for me! Thanks! I get `nvm not found` when trying to load ubuntu profile though... weird. I think this is unrelated.

Reply:
Yeah, it seems like `.bashrc` should be loaded first before Ubuntu's `.profile`. After zsh is being used, does it work as expected?